.github/actions/nano/setup-pytorch-env/action.yml

```diff
@@ -81,7 +81,7 @@ runs:
 
         # required by Ray test
         pip install google-api-core==2.8.2
-        pip install ray[default]==1.11.0 prometheus_client==0.13.0
+        pip install ray[default]==1.13.0 prometheus_client==0.13.0
```

Comment:
I think it's all right to do this for now, but we should bear in mind that orca only support 1.9.2 and there is a gap between nano's ray and orca's ray.